pkgs/development/python-modules/lineedit/default.nix

```diff
@@ -0,0 +1,26 @@
+{ lib, buildPythonPackage, fetchFromGitHub, pygments, six, wcwidth
+, pytestCheckHook, pyte, ptyprocess, pexpect }:
+
+buildPythonPackage rec {
+  pname = "lineedit";
+  version = "0.1.6";
+
+  src = fetchFromGitHub {
+    owner = "randy3k";
+    repo = pname;
+    rev = "4262e9852aa50b09ff3f0569e1840b003ed00584";
```

Comment:
```suggestion
    rev = "releases/tags/v${version}";
```

Reply:
This requires `rev = v${version}` to work. I've changed everything accordingly.